.github/workflows/unittests.yml

```diff
@@ -2,51 +2,49 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       max-parallel: 10
       fail-fast: false
       matrix:
+        os: [ubuntu-latest, ]
```

Comment:
What do you think to add `mac-latest` too?

Reply:
Maybe in the same as windows...